modules/kubernetes/datadog/main.tf

```diff
@@ -2,6 +2,7 @@
   * # Datadog
   *
   * Adds [Datadog](https://github.com/DataDog/helm-charts) to a Kubernetes cluster.
+  * This module is built to only gather application data
```

Comment:
Please extend documentation to describe what this means. We have customers using datadog for the whole cluster and should be clear that they can't use this module for that.